plugins/net_plugin/net_plugin.cpp

```diff
@@ -796,11 +795,11 @@ namespace eosio {
       void handle_message( const request_message& msg );
       void handle_message( const sync_request_message& msg );
       void handle_message( const signed_block& msg ) = delete; // signed_block_ptr overload used instead
-      void handle_message( const block_id_type& id, signed_block_ptr msg );
+      void handle_message( const block_id_type& id, signed_block_ptr ptr );
```

Comment:
`ptr` is not a very descriptive name.

Reply:
Neither is `msg`. Just made it match the definition.